testng/testng-build.gradle.kts

```diff
@@ -51,11 +51,12 @@ tasks.mergedJar {
             "Bundle-Description" to project.description,
             "Bundle-Version" to project.version.toString().removeSuffix("-SNAPSHOT"),
             "Import-Package" to """
-                bsh.*;version="[2.0.0,3.0.0)";resolution:=optional,
-                com.beust.jcommander.*;version="[1.7.0,3.0.0)";resolution:=optional,
-                com.google.inject.*;version="[1.2,1.3)";resolution:=optional,
-                org.yaml.*;version="[1.6,2.0)";resolution:=optional,
-                *;resolution:=optional
+                com.beust.jcommander;version="1.7.0",
+                org.slf4j;version="2.0",
+                com.google.inject;version="1.2.0";resolution:=optional,
```

Comment:
better to use newer version like `5.0.0` for guice to align with testng-core https://github.com/testng-team/testng/blob/master/testng-core/testng-core-build.gradle.kts#L36

Reply:
guice 4.1.0 exports `com.google.inject;version="1.4"` as well, so 1.4 is good enough here